src/core_plugins/kibana/public/management/sections/objects/components/objects_table/objects_table.js

```diff
@@ -0,0 +1,315 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { flattenDeep } from 'lodash';
+import { Header } from './components/header';
+import { Flyout } from './components/flyout';
+import { Relationships } from './components/relationships';
+import { Table } from './components/table';
+
+import { EuiSpacer, EuiHorizontalRule, Query } from '@elastic/eui';
+import { retrieveAndExportDocs } from '../../lib/retrieve_and_export_docs';
```

Comment:
Might want to consider creating `lib/index.js` that exports everything, so we can do one import for all of these.

```
import {
  retrieveAndExportDocs,
  scanAllTypes,
  ...
} from '../../lib';
```